### ISSUE TITLE
Make good design of Registration page

### ISSUE THREAD

**Issue by vilsijain** [vilsi12/Red-Hat-Mobil-App#6]
Check the video https://youtu.be/t1Fo3KUx1_k
 before Starting i have told how to start the app

**Comment by harshsinghs1058** [comment]
Can you share the discord link

**Comment by vilsi12** [comment]
It is in youtube link description

**Comment by kmryashasvi** [comment]
Just design or you need a source code too????????

**Comment by vilsi12** [comment]
actually source code

**Comment by pranavi79** [comment]
I have a sound knowledge of flutter+figma ans previously worked on several projects designing, can I work on this issue?

**Comment by vilsi12** [comment]
Yes Make a good design and check youtube video which i hava made

**Comment by pranavi79** [comment]
On it! thank you so much!

**Comment by vilsi12** [comment]
checked the video and follow
